apps/ecommerce/frontend/src/hooks/useExternalResource.tsx

```diff
@@ -19,7 +20,7 @@ const useExternalResource = (resource?: ExternalResourceLink) => {
 
   const hydrateExternalResource = useCallback(
     async (resource: ExternalResourceLink) => {
-      const url = new URL(`${config.backendApiUrl}/api/resource`);
+      const url = new URL(`${config.backendApiUrl}/shopify/resource`);
```

Comment:
Should we be using the `baseUrl` from the instance parameters here instead of pulling from this config? The `baseUrl` is what I used in the `ConfigPage` component (e.g. https://github.com/contentful/apps/blob/master/apps/ecommerce/frontend/src/components/Config/ConfigPage.tsx#L75). This way, you don't have to hardcode `shopify` in the url

Reply:
Oh cool, good call! Will switch it to `baseUrl` 👌

Reply:
Updated to use `baseURL` in latest commit.